test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -2362,6 +2395,7 @@ public class Microsoft.AspNet.OData.Query.ODataQueryOptions {
 	public virtual object ApplyTo (object entity, ODataQuerySettings querySettings)
 	public virtual System.Linq.IQueryable ApplyTo (System.Linq.IQueryable query, ODataQuerySettings querySettings, AllowedQueryOptions ignoreQueryOptions)
 	public virtual object ApplyTo (object entity, ODataQuerySettings querySettings, AllowedQueryOptions ignoreQueryOptions)
+	public OrderByQueryOption GenerateStableOrderByQueryOption ()
```

Comment:
>OrderByQueryOption [](start = 8, length = 18)

Should this be public virtual? Might someone want to override with a custom implementation?
Also, can name be simplified (i.e., "GenerateStableOrder").